src/chrome/content/rules/Deutsche-Rentenversicherung.de.xml

```diff
@@ -0,0 +1,11 @@
+<ruleset name="Deutsche-Rentenversicherung.de">
+	<target host="www.deutsche-rentenversicherung.de" />
+	<target host=    "deutsche-rentenversicherung.de" />
+	<target host="www.deutsche-rentenversicherung-bund.de" />
+	<target host=    "deutsche-rentenversicherung-bund.de" />
+	<target host="www.drv-bund.de" />
+	<target host=    "drv-bund.de" />
+
+	<rule from="^https?://(?:www\.)?(?:deutsche-rentenversicherung\.de|deutsche-rentenversicherung-bund\.de|drv-bund\.de)/"
```

Comment:
The \.de is common to all 3 capture groups... You might aswell just put it after

"^https?://(?:www\.)?(?:deutsche-rentenversicherung|deutsche-rentenversicherung-bund|drv-bund)\.de/"

But you can simplify it further..

"^https?://(?:www\.)?(?:deutsche-rentenversicherung(-bund)?|drv-bund)\.de/"